frontend/src/pages/Navbar/RightNavbarNavigation.js

```diff
@@ -32,13 +32,15 @@ const RightNavbarNavigations = ({
   const searchDisabled = history.location.pathname !== "/projects";
   return (
     <div style={styles.container}>
-      <ProjectSearch
-        searchBarDisplayed={searchBarDisplayed}
-        searchTerm={searchTerm}
-        searchDisabled={searchDisabled}
-        storeSearchBarDisplayed={storeSearchBarDisplayed}
-        storeSearchTerm={storeSearchTerm}
-      />
+      {!searchDisabled ? (
+        <ProjectSearch
+          searchBarDisplayed={searchBarDisplayed}
+          searchTerm={searchTerm}
+          searchDisabled={searchDisabled}
```

Comment:
delete searchDisabled from props
also delete it in ProjectSearch
./Common/Searchbar keeps the prop instead